### ISSUE TITLE
Make it possible to copy record.

### ISSUE THREAD

**Issue by seabelis** [internetarchive/openlibrary#2137]
## Is your feature request related to a problem? Please describe.
Frequently there are cases where I want to add an edition that is almost an exact copy of an existing edition record. It would be useful to have the ability to make an instant duplicate of a record so that one can copy the details and simply modify the differences.

## Describe the solution you'd like
Ideally, a button "Make Copy" would bring up a list of populated fields with checkboxes for user to identify which fields to copy to the new record. (Internet Archive ID always excluded, of course.)

## Additional context
For example, this record exists for an IA copy: https://openlibrary.org/books/OL2377707M/The_Book_of_Questions
This also exists in IA, but with no record on OL: https://archive.org/details/bookofquestions00stoc (now added)
All details are exactly the same, the only difference being the IA identifier.

**Comment by tfmorris** [comment]
We might want this feature restricted to certain sets of users since, in general, we're trying to eliminate duplicates as opposed to create them.

In this particular use case, it seems like a better solution might be an "Import from IA" function. Of course, ideally, it shouldn't require user intervention to get OL records created for IA books.

**Comment by seabelis** [comment]
Yes. It should only be used in special cases. A restriction would be appropriate.

**Comment by mekarpeles** [comment]
I think the goal here is there are some editions with multiple `ocaid`s which we want to support

**Comment by hornc** [comment]
This seems to have been closed by mistake.
I recently noticed a `clonebook` endpoint in the code:

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/openlibrary/code.py#L240

Perhaps this can be used for this feature -- needs investigation

**Comment by hornc** [comment]
@cdrini didn't one of your recent PRs provide this?
